api/v1/installation_types.go

```diff
@@ -135,7 +135,9 @@ type PreferredNodeAffinity struct {
 	PreferredDuringSchedulingIgnoredDuringExecution []v1.PreferredSchedulingTerm `json:"preferredDuringSchedulingIgnoredDuringExecution,omitempty"`
 }
 
-// ComponentName CRD enum
+// ComponentName represents a single component.
+//
+// One of: Node, Typha, KubeControllers, ECKOperator
```

Comment:
This is wrong now because ECKOperator is now in its own type.

Reply:
According to the enum itself it's still valid (see line 147 below) - should I remove that?